intro.tex

```diff
@@ -12,45 +12,78 @@ \subsection{Motivation for an Early Science Program}  \label{ssec:motivation}
 
 The Early Science program is motivated by the desire to:
 \begin{itemize}
-\item enable high-impact science as early as possible;
-\item provide early access to both static-sky and time-domain science-ready data products to support the community to prepare in advance of the first survey data release;
+\item enable high-impact science with LSST as early as possible;
```

Comment:
Should this be "with Rubin Observatory data"? Asking because DP1 and DP2 are Rubin data before the start of the 10-year LSST.

Reply:
Both actually as DR1 is "Early Science" -- good catch